.konflux/operator/rpms.lock.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
The problem is that Mint Maker won't consider rpm version bumps or docker file updates for base images because these are not root files.  I'm still unsure what is the alternative but I believe for NROP we shouldn't have an issue with that considering must-gather and container image can use the same base images, thus I think these can be moved to the source root (not the most clean way but better to mintmaker do chore updates)

Reply:
Yes I am aware of this current gap and addressed it in the [README section](https://github.com/openshift-kni/numaresources-operator/blob/454771bacb3ca4b3eb469a90e12bdf35281074e0/.konflux/README.md#update--rpms )

If we cannot get automatic updates for sub dirs then we can reconsider extraction to root again.